extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -198,6 +198,11 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 				return;
 			}
 
+			if (context.opts.adMix1Enabled && context.targeting.hasFeaturedVideo) {
```

Comment:
Can we keep it here?
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/adContext.js#L102